scapy/arch/__init__.py

```diff
@@ -16,6 +17,7 @@
 import scapy.config
 from scapy.pton_ntop import inet_pton
 from scapy.data import *
+from scapy.modules.six.moves import map
```

Comment:
I think that it will be nice to rewrite the map calls using list comprehensions. I can do that in another PR if you want.